src/baseline.c

```diff
@@ -501,7 +501,7 @@ s8 lesq_solve_raim(u8 num_dds_u8, const double *dd_obs,
     if (n_used) {
       *n_used = 0;
     }
-    return -3;
+    return -5;
```

Comment:
Yeah I'm not a fan of the magic constants either.

It should also print out the meaning of the value too, so perhaps a function to return an string from the enum will be good.